docker/test-server/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+cd /app
+exec python manage.py runserver 0.0.0.0:3735
```

Comment:
Check the readme, this is not the recommended way to run it anymore (not sure it even works). You need to use uvicorn.

Reply:
Ahh, you did it because it was 0.7.0. Yeah, please keep it at latest and then move to uvicorn.